components/volunteer-form/volunteer-form.html

```diff
@@ -72,13 +72,31 @@ <h1 title>Volunteer Registration</h1>
         value="{{data.school}}">
       </school-input>
 
-      <paper-input
+      <paper-dropdown-menu
         required
         auto-validate
+        horizontal-align="left"
+        error-message="Required"
+        label="What faculty are you in?"
+        selected-item-label="{{data.faculty}}">
+        <paper-listbox slot="dropdown-content">
+          <paper-item>Arts</paper-item>
+          <paper-item>Business</paper-item>
+          <paper-item>Science</paper-item>
+          <paper-item>Engineering</paper-item>
+          <paper-item>Forestry</paper-item>
```

Comment:
Faculty seems like it's missing a lot compared to major.

I personally think it'd be better to scrape all specializations off of https://courses.students.ubc.ca/cs/main?pname=spec and then use a autocomplete input similar to how we're doing schools.

Reply:
@luceliu

Reply:
this would be cool, but at this point it's also pretty late in the game, and not sure who else would be comfortable implementing the scraping. unless some miracle happens before/after your midterm, let's stick with faculty? @d4l3k